frontend/packages/ceph-storage-plugin/src/utils/block-pool.tsx

```diff
@@ -229,11 +229,11 @@ export const checkRequiredValues = (
   isPoolManagementSupported: boolean,
 ): boolean => !poolName || !replicaSize || (isPoolManagementSupported && !volumeType);
 
-export enum FooterPrimaryActions {
-  CREATE = 'Create',
-  DELETE = 'Delete',
-  UPDATE = 'Save',
-}
+export const FooterPrimaryActions = (t: TFunction) => ({
+  CREATE: t('ceph-storage-plugin~Create'),
+  DELETE: t('ceph-storage-plugin~Delete'),
+  UPDATE: t('ceph-storage-plugin~Save'),
+});
```

Comment:
Strongly against mixing testing and translations.

Reply:
The tests are failing because of `checkErrors` on the console browser. we won't be able to merge it.